test/Scripts.Integration.Test/Scripts/RuntimeOptions.cs

```diff
@@ -9,36 +10,8 @@ public override void Configure(SentryUnityOptions options)
     {
         Debug.Log("Sentry: RuntimeOptions::Configure() called");
 
-        options.Dsn = string.Format("http://publickey@{0}:8000/12345",
-#if UNITY_ANDROID
-            "10.0.2.2");
-#elif UNITY_WEBGL
-            "127.0.0.1");
-#else
-            "localhost");
-#endif
-
-        Debug.LogFormat("Sentry: Setting options.Dsn = {0}", options.Dsn);
-
-        options.AttachScreenshot = true;
-        options.Il2CppLineNumberSupportEnabled = true;
-        options.Debug = true;
-        options.DiagnosticLevel = SentryLevel.Debug;
-        options.TracesSampleRate = 1.0d;
-        options.PerformanceAutoInstrumentationEnabled = true;
-
-        options.CreateHttpMessageHandler = () => SmokeTester.t;
-        SmokeTester.CrashedLastRun = () =>
-        {
-            if (options.CrashedLastRun != null)
-            {
-                return options.CrashedLastRun() ? 1 : 0;
-            }
-            return -2;
-        };
-
-        // If an ANR triggers while the smoke test runs, the test would fail because we expect exact order of events.
-        options.DisableAnrIntegration();
+        // Make sure the config is still called and the DSN has been set
+        Assert.IsFalse(string.IsNullOrWhiteSpace(options.Dsn));
```

Comment:
oh actually now we get an assert here, while before we just wrote some setting to assert later?

Reply:
This is the same as the `BuildTimeOptions` above. The options used for smoke testing are consolidated in the new `OptionConfiguration` callback and here we're making sure those options are still getting set and the old way is run last.